Documentation/Books/Manual/Architecture/DeploymentModes/Cluster/Architecture.md

```diff
@@ -95,9 +101,31 @@ which are suitable for different usage scenarios:
     latency. Essentially, this moves some of the database distribution
     logic to the machine where the client runs.
 
-As you acn see, the _Coordinator_ layer can be scaled and deployed independently
+As you can see, the _Coordinator_ layer can be scaled and deployed independently
 from the _DBServer_ layer.
 
+{% hint 'warning' %}
+It is a best practice and a recommended approach to not run _Agent_ instances
+on the same machines where _DBServer_ instances are running.
+
+When deploying using the tool [_Starter_](../../../Deployment/ArangoDBStarter/README.md)
+this can be achieved by using the options `--cluster.start-dbserver=false` and
+`--cluster.start-coordinator=false` on the first three machines where the _Starter_
+is started, if the desired _Agency_ _size_ is 3, or on the first 5 machines
+if the desired _Agency_ _size_ is 5.
+{% endhint %}
+
+{% hint 'info' %}
+The difference instances part of a single Cluster are supposed to be run in the same
+_Data Center_ (DC), with reliable and high-speed network connection between
+all the machines participating to the Cluster.
+
+Multi-datacenter Clusters, where the entire structure and content of a Cluster located
+in a specific DC is replicated to others Clusters located in different DCs, are
+possible as well. See [Datacenter to datacenter replication](../DC2DC/README.md)
+(DC2DC) for further details.
```

Comment:
Is this really officially supported? It sounds like you could use DC2DC to create a distributed cluster, which it is not. Also, do we really support 1:N datacenters in DC2DC? In a discussion with Omar our understanding was that you can use it for 1:1 async replication between two clusters, in a single direction (one way).